spec/controllers/objects_controller_spec.rb

```diff
@@ -53,19 +78,19 @@
     end
 
     context 'when bad parameter passed in' do
-      it 'returns a 500 response code' do
+      it 'returns a 400 response code' do
```

Comment:
i'd blame the calling client (not the server) for this.  unclear to me if `400` is exactly right, but `4xx` feels more right to me than `5xx` here.